pkg/client/portforward/portforward_test.go

```diff
@@ -253,6 +249,20 @@ func TestGetListener(t *testing.T) {
 		},
 	}
 
+	// On some linux systems, ::1 does not resolve to localhost but to localhost6 or
+	// ip6-localhost. To make the test case portable, we need to do a reverse lookup on ::1 and
+	// trying to bind a port with the name.
+	names, err := net.LookupAddr("::1")
+	if err == nil && len(names) > 0 {
```

Comment:
Silent skip on a failure seems bad. Why not fail or at least log on a failed lookup?